.jenkins/pytorch/win-test-helpers/run_python_nn_smoketests.py

```diff
@@ -3,9 +3,10 @@
 from __future__ import print_function
 
 import subprocess
+import os
 
 
-TESTS = [
+GPU_TESTS = [
```

Comment:
What about splitting them into `GPU_ONLY_TESTS` and `CPU_TESTS` so that you don't have to repeat some of the items?

Reply:
Ok